packages/services/f-stat-client/README.md

```diff
@@ -0,0 +1,109 @@
+<div align="center">
+
+# f-stat-client
+
+<img width="125" alt="Fozzie Bear" src="../../../bear.png" />
+
+Javascript HTTP client for publishing stats to ElasticSearch
+
+</div>
+
+---
+
+[![npm version](https://badge.fury.io/js/%40justeat%2Ff-stat-client.svg)](https://badge.fury.io/js/%40justeat%2Ff-stat-client)
+[![CircleCI](https://circleci.com/gh/justeat/fozzie-components.svg?style=svg)](https://circleci.com/gh/justeat/workflows/fozzie-components)
+[![Coverage Status](https://coveralls.io/repos/github/justeat/f-stat-client/badge.svg)](https://coveralls.io/github/justeat/f-stat-client)
+[![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-stat-client/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-stat-client?targetFile=package.json)
+
+This client abstracts away the complexity of publishing stats to ElasticSearch, such as API timings so you can then graph on these results.  It also provides a standardised approach for to follow that can be used in a suite of features and components, allowing you to use and report in a generic way.
+
+## Benefits (Now)
+- Simple publish method that hides away the complexities of making posts requests to ElasticSearch
+- Sensible defaults, with the ability to override all
+- If you run ElasticSearch locally in a Docker Container and use the host `localhost` then you can see you results instantly (see https://www.elastic.co/guide/en/elasticsearch/reference/7.0/docker.html & https://elasticvue.com/)
+- Built-in mock ability by providing a mock on the constructor (see https://github.com/elastic/elasticsearch-js-mock and an example below)
+
+## Benefits (Soon)
+- _encapsualted batch publishing_
+- _encapsualted sample publishing_
+- _encapsualted authentication_
+<hr></br>
+
+## Usage
+</br>
+
+### *Installation*
+
+Install the module using npm or Yarn:
+
+```sh
+yarn add @justeat/f-stat-client
+```
+### *Initialisation/Construction e.g.*
+```js
+import StatClient from '@justeat/f-stat-client';
+
+const client = new StatClient('http://localhost', 9200, 'uk', 'checkoutWeb');
+
+```
+### *How to use*
+```js
+
+await client.publish('GET', '/search', 200, 611);
```

Comment:
I think this is good for now to work for our prototype - but I suspect we may want to change this later to support accepting an object with parameters - as that will ensure we cover other use cases like search, and any other stats we want to push